cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -93,6 +93,7 @@ current state towards the desired state. Examples of controllers that ship with
 Kubernetes today are the replication controller, endpoints controller, namespace
 controller, and serviceaccounts controller.`,
 		Run: func(cmd *cobra.Command, args []string) {
+			Run(s)
```

Comment:
You need to move `verflag.PrintAndExitIfRequested()` here

Reply:
> You need to move verflag.PrintAndExitIfRequested() here as the first line in the func

done